buildSrc/src/main/kotlin/plugin/AppConfigurer.kt

```diff
@@ -83,6 +90,12 @@ class AppConfigurer {
     configureDependencies(target)
   }
 
+  private fun setNameForNightlyUniversalApk(): String =
+    "kiwix-universal-${getCurrentDate()}.apk"
```

Comment:
@MohitMaliDeveloper Currently it's naming `kiwix-universal-$date.apk` . As @kelson42 mentioned it should be `kiwix-android_multiarch_$date`. But if @kelson42 agrees with the naming then I'm good too. Otherwise, everything looks good to me!

Reply:
Actually, universal is the proper name. I agree with @MohitMaliFtechiz